packages/create-sitecore-jss/src/templates/react/scripts/bootstrap.js

```diff
@@ -1,32 +1,21 @@
-require('dotenv').config();
-const { constants } = require('@sitecore-jss/sitecore-jss-react');
-const configGenerator = require('./generate-config');
-const chalk = require('chalk');
-
 /*
   BOOTSTRAPPING
   The bootstrap process runs before build, and generates JS that needs to be
   included into the build - specifically, the component name to component mapping,
   and the global config module.
 */
 
-const disconnected = process.env.JSS_MODE === constants.JSS_MODE.DISCONNECTED;
-
-if (disconnected && process.env.REACT_APP_FETCH_WITH === constants.FETCH_WITH.GRAPHQL) {
-  throw new Error(chalk.red("GraphQL requests to Dictionary and Layout services are not supported in disconnected mode."))
-}
+/**
+ * PLUGINS GENERATION
+ */
+require('./generate-plugins');
 
-/*
-  CONFIG GENERATION
-  Generates the /src/temp/config.js file which contains runtime configuration
-  that the app can import and use.
-*/
-const port = process.env.PORT || 3000;
-const configOverride = disconnected ? { sitecoreApiHost: `http://localhost:${port}` } : null;
+/**
+ * CONFIG GENERATION
+ */
+require('./generate-config');
```

Comment:
Would env values for config be read anywhere?
In nextjs we set initial values from env and set the missing values from package.json and scjssconfig. 
And would it be better to make bootstrap into plugin too, for parity with nextjs?

Reply:
I added env variables for default config
but bootstrap doesn't have plugins (even in nextjs sample app) and I think that we don't really need it since we just need to call a few scripts, maybe let's "pluginize" it if it will be more complex? (like component builder generation)